frontend/src/config/cambodia/layers.json

```diff
@@ -1713,6 +1713,14 @@
       "status": {
         "type": "text",
         "dataTitle": "Form status"
+      },
+      "submission_id": {
```

Comment:
Maybe metadata should just keep their initial name and be a bit more lenient in the netadata type? Wdyt?
No strong feelings, open to suggestions.

cc @laurentS interested in your input as well

Reply:
I transform submission_id in dmpSubmissionId to use camel case in the typescript code, and add also dmp prefix to be more specific about what is submission id. But if you rather like to use submission_id directly it is ok for me.

"be a bit more lenient in the metadata type" do you suggest to add a type to control and raise an error if it is a number rather than a string for instance? It is possible but it appears a little overkill for me. If required I can add this control.

Reply:
No I actually meant "do not create a type for metadata and return all additional info that's not "data". This way people can simply decide to use the extra data in their logic, or not. :)
I am ok with the current implementation as well

Reply:
So thinking about it a bit more we could actually revamp this a bit to also take into account the "do_not_display" behavior by making the dataTitle optional.

Then:
Option 1 - keys in `feature_info_props` would become data and all additional info would become metadata
Option 2 - keys in `feature_info_props` accept a "metadata" flag to split info between data and metadata like you're doing now.

Wdyt?

Reply:
I think option 2 is better to covert the case when we want both to display the data in the popup and use the data in the metadata while remaining generic. But I bring a small simplification in the layer.json to simplify metadata object.